docs/swagger/spec.yml

```diff
@@ -101,7 +101,9 @@ x-resources:
   - name: Admin
     description: |
       Platforms, Organizations, and Teams can be used to manage users and visibility of projects, analyses, and scenes. Most of these functions will only be available to uses who are administrating groups.
-
+  - name: Stac
```

Comment:
is it Stac, or STAC, ST.A.C, or S.T.A.C.?

Reply:
STAC